vita3k/modules/SceRegistryMgr/SceRegMgr.cpp

```diff
@@ -40,30 +42,34 @@ EXPORT(int, sceRegMgrGetInitVals) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceRegMgrGetKeyBin) {
-    TRACY_FUNC(sceRegMgrGetKeyBin);
-    return UNIMPLEMENTED();
+EXPORT(int, sceRegMgrGetKeyBin, const char *category, const char *name, void *buf, SceSize bufSize) {
+    TRACY_FUNC(sceRegMgrGetKeyBin, category, name, buf, bufSize);
+    memcpy(buf, regmgr::get_str_value(emuenv.regmgr, category, name).c_str(), bufSize);
+
+    return 0;
 }
 
-EXPORT(int, sceRegMgrGetKeyInt, const char *category, const char *name, int *buf) {
+EXPORT(int, sceRegMgrGetKeyInt, const char *category, const char *name, SceInt32 *buf) {
     TRACY_FUNC(sceRegMgrGetKeyInt, category, name, buf);
-    if ((std::string(category) == "/CONFIG/PSPEMU") && (std::string(name) == "emu_list_flag")) {
-        STUBBED("Stubbed");
-        *buf = 1;
```

Comment:
What was this used for? Will it break some game?

Reply:
nop, is my old stub, now i using real value

Reply:
Well I was wondering since there is no /CONFIG/PSPEMU in get_category_and_name_by_id

Reply:
normal, module regmgr no using id, it give directly category and name
is module  ForGame and ForSdk using id
and for now, i have register in this function all id i have see called
i have add log, for can adding next in futur is view